src/Components/test/testassets/Components.TestServer/RazorComponents/Pages/EnhancedNav/PageWithServerRenderModeCanNavigate.razor

```diff
@@ -0,0 +1,15 @@
+﻿@page "/nav/render-mode-server-can-navigate"
+@inject NavigationManager Nav
+@attribute [RenderModeServer]
```

Comment:
If possible this would make it a more robust test:

```suggestion
@attribute [RenderModeServer(Prerender = false)]
```

... because otherwise maybe the test would pass falsely if Selenium tries to navigate before interactivity has actually started. With `Prerender = false`, we know the button won't exist at all until interactivity has started, so we can be sure it's a valid result.

Reply:
Oh no forget that, if it's not interactive then `@onclick` wouldn't run anyway, so it was already completely valid!